daemon/prune.go

```diff
@@ -122,7 +124,7 @@ func (daemon *Daemon) VolumesPrune(ctx context.Context, pruneFilters filters.Arg
 		select {
 		case <-ctx.Done():
 			logrus.Warnf("VolumesPrune operation cancelled: %#v", *rep)
-			return ctx.Err()
+			return errPruneCancelled
```

Comment:
I don't think it's right to return a cancelled error here since cancellation is just one reason why the context would be done.